recipes/python-ecmwf_grib/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
```

Comment:
We should comment that this is just ecmwf_grib/recipe/build.sh with the ```--enable-python``` turned on.